flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCInputFormatTest.java

```diff
@@ -101,6 +105,33 @@ public void testIncompleteConfiguration() throws IOException {
 	}
 
 	@Test
+	public void defaultFetchSizeIsUsedIfNotConfiguredOtherwise() throws SQLException {
+		jdbcInputFormat = JDBCInputFormat.buildJDBCInputFormat()
+			.setDrivername(DRIVER_CLASS)
+			.setDBUrl(DB_URL)
+			.setQuery(SELECT_ALL_BOOKS)
+			.setRowTypeInfo(ROW_TYPE_INFO)
+			.finish();
+		jdbcInputFormat.openInputFormat();
+		assertThat(jdbcInputFormat.getStatement().getFetchSize(), equalTo(1));
```

Comment:
The idea was to make sure that existing behavior is not broken. I have made the dependency on the fact that derby is used in this test explicit.